io.openems.edge.controller.symmetric.reactivepowervoltagecharacteristic/bnd.bnd

```diff
@@ -1,7 +1,10 @@
-Bundle-Name: OpenEMS Edge Controller Symmetric Reactive-Power Voltage Characteristics
```

Comment:
Should we move all those Controllers to the `io.openems.edge.controller.ess` namespace?